docs/userGuide/reusingContents.md

```diff
@@ -101,6 +101,13 @@ As mentioned in _User Guide: MarkBind Syntax_, you can use built-in variables to
 <p/>
 
 
+##### Variables: Defaults
+
+You can specify a default value for a variable, which is displayed when the variable is not specified in `variables.md`. This is done by adding `or (defaultValue)` within the curly braces.
+
+<code>My name is {<span></span>{ name or "Anonymous" }}.</span></code>
```

Comment:
```
<div class="indented">

{{ icon_example }} If `name` is not declared in `variables.md`:<br>
<code>My name is {<span></span>{ name or "Anonymous" }}.</span></code> {{ icon_arrow_right }} My name is Anonymous.
</div>
```

Reply:
Why is there `</span>` right before `</code>`?

Reply:
> Why is there `</span>` right before `</code>`?

My bad, didn't spot the error. The `</span>` is not necessary. Edited the original post.